packages/wp-checkout/src/components/checkout-step.js

```diff
@@ -55,19 +60,31 @@ CheckoutStep.propTypes = {
 	stepNumber: PropTypes.number.isRequired,
 	title: PropTypes.string.isRequired,
 	finalStep: PropTypes.bool,
-	summary: PropTypes.bool,
+	stepSummary: PropTypes.node,
```

Comment:
I think it's a good idea to make at least `stepContent` required, and maybe both of them (if we make either optional we should only display it if it exists, otherwise we might (?) end up displaying "undefined").

Reply:
That's true. The content should definitely be required. The summary could potentially be empty (I'm thinking the billing step).